pkg/corerp/renderers/gateway/render.go

```diff
@@ -366,6 +369,7 @@ func MakeRoutesHTTPProxies(ctx context.Context, options renderers.RenderOptions,
 							},
 						},
 						PathRewritePolicy: pathRewritePolicy,
+						EnableWebsockets:  true,
```

Comment:
Are there any downsides to this? what do the contour docs say?

Reply:
Here is the link to the documentation: https://projectcontour.io/docs/main/config/websockets/#websockets.

Now that I think about this again, I think we will need to add a property to our Gateway resource so that setting **enableWebsockets** to true all the time doesn't affect other routes that don't need **websockets** enabled.